packages/paste-core/primitives/combobox/__tests__/useMultiSelectPrimitive.test.tsx

```diff
@@ -0,0 +1,1720 @@
+import * as React from 'react';
+
+import {act, renderHook} from '@testing-library/react-hooks';
+import {render, fireEvent, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+
+import type {FireObject} from '@testing-library/react';
+import type {
+  UseMultipleSelectionGetSelectedItemPropsOptions,
+  UseMultipleSelectionGetDropdownProps,
+} from '@twilio-paste/dropdown-library';
+
+import {useMultiSelectPrimitive, useComboboxPrimitive} from '../src';
+import type {UseComboboxPrimitiveProps, UseMultiSelectPrimitiveProps} from '../src';
+
+const {stateChangeTypes} = useMultiSelectPrimitive;
+
+jest.mock('@twilio-paste/dropdown-library', () => {
+  const {default: all, ...rest} = jest.requireActual('@twilio-paste/dropdown-library');
+
+  return {
+    ...rest,
+    default: {
+      ...all,
+      generateId: () => 'test-id',
+      useGetterPropsCalledChecker: () => all.noop,
+    },
+  };
+});
```

Comment:
Just for my knowledge, is this something they do in their tests, or genuinely because it couldn't find the dependency?

Reply:
This is copied from their unit tests. For some reason it looks like this utility function is [only defined when the environment is production](https://github.com/downshift-js/downshift/blob/1c7698080805e05f5180dd223cbc17932a85fd42/src/hooks/utils.js#L395). 😅